src/LEGO.AsyncAPI.Readers/AsyncApiReaderSettings.cs

```diff
@@ -18,9 +18,14 @@ public enum ReferenceResolutionSetting
         DoNotResolveReferences,
 
         /// <summary>
-        /// Resolve internal component references and inline them.
+        /// Resolve all references and inline them.
         /// </summary>
-        ResolveReferences,
+        ResolveAllReferences,
```

Comment:
Maybe this should stay as just `ResolveReferences` so as to avoid making a breaking change.

Reply:
I think we should try to keep breaking changes to a minimum at least.
At this point it means "resolve internal references" so the change might be necessary, but should then be a complete breaking change to not catch consumers doing things they don't intend

Reply:
Just to clarify are you suggesting to change `ResolveReferences` -> `ResolveInternalReferences` and then introduce a new `ResolveAllReferences` for both?

Rather than updating functionality of `ResolveReferences` to do both and introducing `ResolveInternalReferences` for users to move to?

I'm fine with either just wanted to confirm your suggestion.

Reply:
@VisualBean updated! :)